common/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -1692,6 +1695,186 @@ class BackendTest {
         assertThat(calledDelay).isEqualTo(Delay.LONG)
     }
 
+    // endregion
+
+    // region getProductEntitlementMappings
```

Comment:
We should also split these tests into multiple files, but will leave these refactors for another time.